api/index.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+require_once('../vendor/autoload.php');
+require_once('./services/ConfigService.class.php');
+require_once('./dao/UserDao.class.php');
+require_once('./routes/AuthRoutes.php');
+require_once('./routes/PostRoutes.php');
+require_once('./services/AuthService.class.php');
+$SetupDataBase = require_once('./services/SetupDataBase.class.php');
+$SetupDataBase->createUserTable();
+$SetupDataBase->createPostTable();
```

Comment:
this can be implemented as one route maybe. so you can add all table creations within that route.